recipes/libwebm/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
You no longer need to include test_v1_package for new recipes: https://github.com/conan-io/conan-center-index/discussions/17406